app/assets/javascripts/components/reviewer/reviewerDashboard.js.jsx

```diff
@@ -4,17 +4,17 @@ class ReviewerDashboard extends React.Component {
     super(props)
 
     this.cohorts = JSON.parse(this.props.cohorts)
-    this.current_cohort = JSON.parse(this.props.current_cohort)
+    if(this.current_cohort){this.current_cohort = JSON.parse(this.props.current_cohort)}
     this.defineCurrentCohort()
-    this.current_cohort.title = this.current_cohort.title + ' (Current Cohort)'
+    if(this.current_cohort){this.current_cohort.title + ' (Current Cohort)'}
     this.user = JSON.parse(this.props.user)
     this.authorization = 'Bearer ' + this.props.authorization
   }
   
   defineCurrentCohort() {
     return this.cohorts.map((cohort, i) => {
-      if (cohort.title == this.current_cohort.title) {
-        cohort.title = this.current_cohort.title + ' (Current Cohort)'
+      if (this.current_cohort && cohort.title === this.current_cohort.title) {
```

Comment:
Out of curiosity, why are we doing equality comparison on `title` instead of `id`?

Reply:
Great question; id is a much better choice.